pkgs/development/python-modules/graphqlclient/default.nix

```
@@ -0,0 +1,24 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "graphqlclient";
+  version = "0.2.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0b6r3ng78qsn7c9zksx4rgdkmp5296d40kbmjn8q614cz0ymyc5k";
+  };
+  checkInputs = [
```

`six` is a run-time [requirement](https://github.com/prisma-labs/python-graphql-client/blob/master/setup.py#L16-L17).

```suggestion

  propagatedBuildInputs = [
```